controllers/helpers.go

```diff
@@ -0,0 +1,77 @@
+package controllers
+
+import (
+	"github.com/aws/aws-sdk-go/service/autoscaling"
+	upgrademgrv1alpha1 "github.com/keikoproj/upgrade-manager/api/v1alpha1"
+	"log"
+	"strconv"
+	"strings"
+)
+
+// getMaxUnavailable calculates and returns the maximum unavailable nodes
+// takes an update strategy and total number of nodes as input
+func getMaxUnavailable(strategy upgrademgrv1alpha1.UpdateStrategy, totalNodes int) int {
+	// Below are the constants set in intstr package
+	// const (
+	//	Int    Type = iota // The IntOrString holds an int.
```

Comment:
Are these comments necessary?

Reply:
These were there already, I moved the method from `rolling_upgrade_controller,go` to `helpers.go` as `rolling_upgrade_controller,go`  grew to 1000+ lines.

I can remove them.

Reply:
removed